### PR TITLE
Updates RNs for z-stream 4.9.56

### PR DESCRIPTION
**Pull request by cbippley**
[OSDOCS-5356](https://issues.redhat.com/browse/OSDOCS-5356):Updates release notes for 4.9.56 z-stream release. 

Version(s):
4.9

Issue:
https://issues.redhat.com/browse/OSDOCS-5356

Link to docs preview:
[Preview](http://file.rdu.redhat.com/~cbippley/RNs-4.9-56/release_notes/ocp-4-9-release-notes.html#ocp-4-9-56)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -3383,3 +3383,27 @@ $ oc adm release info 4.9.55 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-56"]
+=== RHSA-2023:0778 - {product-title} 4.9.56 security update
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:3388]:
```suggestion
=== RHSA-2023:0778 - {product-title} 4.9.56 bug fix and security update
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -3383,3 +3383,27 @@ $ oc adm release info 4.9.55 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.9.56 --pullspecs
+----
+
+[id="ocp-4-9-56-bug-fixes"]
+==== Bug fixes
+
+* Previously, pod failures were artificially extending the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and the certificates are correctly rotated. (link:https://issues.redhat.com/browse/OCPBUGS-5938[*OCPBUGS-5938*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:3404]: just a suggestion, up to you if you want any of it. I was thinking more along the lines of past/present tense here:
```suggestion
* Previously, pod failures artificially extended the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and certificates rotate correctly. (link:https://issues.redhat.com/browse/OCPBUGS-5938[*OCPBUGS-5938*])
```
The only thing I am wondering is if we can use another word for correctly for one of the instances in the last sentences. There are 2 and it trips me up a bit.

**Review by opayne1 — COMMENTED** [review]
Just a couple small things!

**Comment by opayne1** [comment]
Also, I am having trouble accessing your preview, but it looks like it is on my end.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (opayne1) [reply release_notes/ocp-4-9-release-notes.adoc]: Yeah I think that would work. Thanks!

Comment (cbippley) [reply release_notes/ocp-4-9-release-notes.adoc]: @opayne1 Thank you!

**Pull request merged**